distribution/docs/src/main/resources/content/_configuring/search-options.adoc

```diff
@@ -0,0 +1,23 @@
+:title: Configuring Search Options for ${catalog-ui}
+:type: subConfiguration
+:status: published
+:parent: Configuring ${catalog-ui}
+:order: 00
+:summary: Configuring local, historical, and archived Search Options for ${catalog-ui}.
+
+== {title}
+
+${catalog-ui} Gives the user a few options to control what metacards can be searched for.
```

Comment:
✏️ `${catalog-ui} provides a few options to control what metacards may be searched.`